packages/flutter/lib/src/scheduler/ticker.dart

```diff
@@ -12,80 +15,233 @@ import 'binding.dart';
+  static const TickerProvider schedulerTickerProvider = const _DefaultTickerProvider();
+}
+
+class _DefaultTickerProvider implements TickerProvider {
+  const _DefaultTickerProvider();
+
+  @override
+  Ticker createTicker(TickerCallback onTick) => new Ticker(onTick);
+}
+
 /// Calls its callback once per animation frame.
 ///
 /// When created, a ticker is initially disabled. Call [start] to
 /// enable the ticker.
 ///
-/// See also [SchedulerBinding.scheduleFrameCallback].
+/// A [Ticker] can be silenced by setting [muted] to true. While silenced, time
+/// still elapses, and [start] and [stop] can still be called, but no callbacks
+/// are called.
+///
+/// Tickers are driven by the [SchedulerBinding]. See
+/// [SchedulerBinding.scheduleFrameCallback].
 class Ticker {
   /// Creates a ticker that will call [onTick] once per frame while running.
-  Ticker(TickerCallback onTick) : _onTick = onTick;
-
-  final TickerCallback _onTick;
+  ///
+  /// An optional label can be provided for debugging purposes. That label
+  /// will appear in the toString output in debug builds.
+  Ticker(this._onTick, { this.debugLabel }) {
+    assert(() {
+      _debugCreationStack = StackTrace.current;
+      return true;
+    });
+  }
 
   Completer<Null> _completer;
-  int _animationId;
-  Duration _startTime;
+
+  /// Whether this ticker has been silenced.
+  ///
+  /// While silenced, a ticker's clock can still run, but the callback will not
+  /// be called.
+  bool get muted => _muted;
+  bool _muted = false;
+  /// When set to true, silences the ticker, so that it is no longer ticking. If
+  /// a tick is already scheduled, it will unschedule it. This will not
+  /// unschedule the next frame, though.
+  ///
+  /// When set to false, unsilences the ticker, potentially scheduling a frame
+  /// to handle the next tick.
+  set muted(bool value) {
+    if (value == muted)
+      return;
+    _muted = value;
+    if (value) {
+      unscheduleTick();
+    } else if (shouldScheduleTick) {
+      scheduleTick();
+    }
+  }
 
   /// Whether this ticker has scheduled a call to call its callback
   /// on the next frame.
-  bool get isTicking => _completer != null;
+  ///
+  /// A ticker that is [muted] can be active (see [isActive]) yet not be
+  /// ticking. In that case, the ticker will not call its callback, and
+  /// [isTicking] will be false, but time will still be progressing.
+  // TODO(ianh): we should teach the scheduler binding about the lifecycle events
+  // and then this could return an accurate view of the actual scheduler.
+  bool get isTicking => _completer != null && !muted;
+
+  /// Whether time is elapsing for this ticker. Becomes true when [start] is
+  /// called and false when [stop] is called.
+  ///
+  /// A ticker can be active yet not be actually ticking (i.e. not be calling
+  /// the callback). To determine if a ticker is actually ticking, use
+  /// [isTicking].
+  bool get isActive => _completer != null;
```

Comment:
We try to avoid "disabled" since it's a negative and you end up in a bad place where you set things to false to enable them.

"enabled" would be wrong since this a ticker could be inactive but enabled. The enabled/disabled wording would be a closer fit for muted/unmuted, but muted is better there since "disabled" would imply "turned off and not advancing time".